src/FieldLabel/FieldLabel.story.tsx

```diff
@@ -51,3 +51,45 @@ export const WithHint = () => (
     <Input />
   </FieldLabel>
 );
+
+export const WithRangeOfInputComponents = () => (
+  <Flex gap="x3" p="x3" borderStyle="dotted" borderWidth="1px" borderColor="lime">
+    <FieldLabel
+      labelText="Label with HelpText that is required"
+      helpText="I am help text. I can give more details on the input below!"
+      requirementText="(Required)"
+    >
+      <Input />
```

Comment:
Do you know what is the difference between 
```
<FieldLabel
      labelText="Label with HelpText that is required"
      helpText="I am help text. I can give more details on the input below!"
      requirementText="(Required)"
    >
      <Input />
```

and 

```
  <Input
      labelText="Label with HelpText that is required"
      helpText="I am help text. I can give more details on the input below!"
      requirementText="(Required)"
  />
```

Reply:
I am not entirely sure what the purpose of the FieldLabel is. Is it supposed to be used outside of NDS and compose with other form inputs or just internally in NDS?

Reply:
I am not sure either